app.rb

```diff
@@ -0,0 +1,33 @@
+require "bundler/setup"
+require "sinatra/base"
+
+Dir["./lib/**/*.rb"].each{|file| require file}
+
+module Shop
+  PRODUCTS = [
+      Product.new( name: "lolz", price: 12 ),
```

Comment:
spacebars aronund brackets are not needed here (use them for `{}` but not for `()`)